src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -89,19 +86,116 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
         addHighlightSet: true,
       });
       closeStepperModal();
+      const handlePublishTrackEvent = () => {
+        const trackInfo = {
+          stepper_modal: {
+            current_step: currentStep,
+            current_step_position: steps.indexOf(currentStep) + 1,
+          },
+          created_highligh_response_result_data: {
+            card_image_url: transformedHighlightSet.cardImageUrl,
+            is_published: transformedHighlightSet.isPublished,
+            highlight_set_title: transformedHighlightSet.title,
+            highlight_set_uuid: transformedHighlightSet.uuid,
+            highlighted_content_uuids: transformedHighlightSet.highlightedContentUuids,
+          },
+        };
+        sendEnterpriseTrackEvent(
+          enterpriseId,
+          `${EVENT_NAMES.CONTENT_HIGHLIGHTS.STEPPER_STEP_CONFIRM_CONTENT}.publish.clicked`,
```

Comment:
nit: include `.publish_btn.clicked` into the constant name as typically the past-tense verb is core to the event itself.